docs/source/API/core/parallel-dispatch/single.md

```diff
@@ -0,0 +1,84 @@
+# `fence()`
+
+Header File: `Kokkos_Core.hpp`
+
+Usage:
+
+```c++
+In a parallel dispatch statement, execute a lambda once per Thread or Team.
+- Kokkos::single` accepts two policies:
+  - Kokkos::PerTeam restricts execution of the lambda's body to once per team
+  - Kokkos::PerThread restricts execution of the lambda body to once per thread (i.e., to only one vector lane in a thread).
+
+```
```

Comment:
I think the c++ block in this section should contain a 1-2 line call that conveys how it is used.